src/main/java/dk/kb/labsapi/SolrBase.java

```diff
@@ -84,6 +87,24 @@ private SolrClient createClient(YAML conf) {
         return new HttpSolrClient.Builder(fullURL).withInvariantParams(baseParams).build();
     }
 
+    /**
+     * Create a SolrClient without any filters.
+     * @param conf the configuration for the Solr client.
+     * @return a SolrClient ready for use.
+     */
+    private SolrClient createRawClient(YAML conf) {
+        String solrURL = conf.getString(".solr.url");
+        String collection = conf.getString(".solr.collection");
+        String fullURL = solrURL + (solrURL.endsWith("/") ? "" : "/") + collection;
+        String filter = conf.getString(".solr.filter", null);
+
+        ModifiableSolrParams baseParams = new ModifiableSolrParams();
+        baseParams.set(HighlightParams.HIGHLIGHT, false);
+        baseParams.set(GroupParams.GROUP, false);
+        log.info("Creating SolrClient({}) without filters", fullURL);
+        return new HttpSolrClient.Builder(fullURL).withInvariantParams(baseParams).build();
+    }
+
```

Comment:
Good idea to create a raw client that can be used for a specific endpoint like this!